packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -130,6 +149,7 @@ const DevApp: FunctionComponent = () => {
         }}
         debug
         key={testBlockType} // completely reset the state when block type has changed
+        initialEntities={Object.values(blockEntityMap)}
```

Comment:
Will need to have a bit more of a think about this because it means the dummy company data is never going to be used so long as someone wants to provide a starting block entity... maybe we have a prop to either add or don't add the bundled dummy data on top of whatever is provided here. Doesn't need to be figured out in this PR